### DISCUSSION TITLE
How to set up the dialog position?

### DISCUSSION THREAD

**Discussion by michaelwv** [AvaloniaUtils/DialogHost.Avalonia#37] [Q&A] [answered]
Hi.
I'm trying to use the DialogHost.Avalonia.
It'is working fine, opening the dialog centered on the screen.
How can I show the dialog at the bottom of the screen?
Thank you.

**Comment by SKProCH** [comment, marked as answer]
Hello, @michaelwv 

You can set `PopupPositioner`. There is a two predefined positioners - `CenteredDialogPopupPositioner` and `AlignmentDialogPopupPositioner`. You can use a last one to achieve the result:
```xaml
<dialogHostAvalonia:DialogHost ...>
  <dialogHostAvalonia:DialogHost.PopupPositioner>
    <positioners:AlignmentDialogPopupPositioner HorizontalAlignment="Right" VerticalAlignment="Bottom"/>
  </dialogHostAvalonia:DialogHost.PopupPositioner>
```
![image](https://github.com/AvaloniaUtils/DialogHost.Avalonia/assets/29896317/0978ab5c-9600-4a93-809c-7fb4dfc5bd17)

Or you can implement `IDialogPopupPositioner` by yourself.
